dev-haskell/haddock-api/haddock-api-2.24.0.ebuild

```diff
@@ -3,29 +3,30 @@
 
 EAPI=7
 
-CABAL_FEATURES="lib hoogle hscolour profile test-suite" # haddock
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
 inherit haskell-cabal
 
 DESCRIPTION="A documentation-generation tool for Haskell libraries"
 HOMEPAGE="https://www.haskell.org/haddock/"
 SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
 
-LICENSE="BSD"
+LICENSE="BSD-2"
 SLOT="0/${PV}"
-# keep in sync with ghc-8.10
+# requires ghc 8.10
 # KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 IUSE=""
 
 RDEPEND=">=dev-haskell/ghc-paths-0.1.0.9:=[profile?] <dev-haskell/ghc-paths-0.2:=[profile?]
 	>=dev-haskell/haddock-library-1.9.0:=[profile?] <dev-haskell/haddock-library-1.10:=[profile?]
 	>=dev-haskell/xhtml-3000.2.2:=[profile?] <dev-haskell/xhtml-3000.3:=[profile?]
 	>=dev-lang/ghc-8.10:=[profile?] <dev-lang/ghc-8.11:=[profile?]
+	>=dev-lang/ghc-8.10.1:=
 "
 DEPEND="${RDEPEND}
-	>=dev-haskell/cabal-3.0.0.0
+	>=dev-haskell/cabal-3.2.0.0
 	test? ( >=dev-haskell/ghc-paths-0.1.0.12 <dev-haskell/ghc-paths-0.2
 		>=dev-haskell/hspec-2.4.4 <dev-haskell/hspec-2.8
 		>=dev-haskell/quickcheck-2.11 <dev-haskell/quickcheck-2.14 )
 "
-
-PATCHES=("${FILESDIR}"/${P}-ghc-8.10.2.patch )
```

Comment:
Why was this patch dropped? But beyond that I'm not too sure what this commit is for. It's neither adding 2.24.0 to the overlay nor keywording it.

Reply:
This was my clumsy attempt to keyword this version of `haddock-api` in order to let my bumped `hsdev` build.

Instead, I will remove the keywords in the bumped `hsdev` pending `ghc 8.10`

Reply:
actually, I left the `hsdev` with the keywords, but modified my change so it only affects the `Cabal` dependencies and nothing else.